### ISSUE TITLE
Xopowo Budget Request 2024-01-25

### ISSUE THREAD

**Issue by saltyfacu** [yearn/budget#188]
### Scope

Xopowo's overall commitment remains the same as in #168. In short: Xopowo is focused on supporting existing Yearn projects while innovating and shipping new ones.

### Highlights

- Built and launched a flagship UI for Yearn v3.
- yPrisma UI v2 hitting prod in the coming days.
- Gimme (a new Yearn front end) in design phase with the team very excited about its potential.
- Juiced Vaults, a new brand for high reward vaults, built and launched with iterations and improvements to come (not to mention new Vaults).
- Several iterations on what and how to show APRs on V3
- yDaemon: better prices, better performance, better data
- Halfway through code split to make all Yearn dapps easier to maintain.
- The team grew with new devs added and we are still looking to add another dev and UI designer. This was an area of weakness in our last BR and we are happy with the progress here.

**tldr**
The last quarter saw a big push on new launches like v3, yPrisma, and Juiced Vaults. Giving each Yearn project it's own visual brand with bespoke UI design is certainly more labour intensive but we feel definitely worth the effort (with the response to the recent Juiced launch being particularily encouraging for this strategy) and we're excited to share the yPrisma UI refresh soon. 

Next quarter we are excited to build Gimme which we see as a big opportunity for Yearn and strategically significant on multiple fronts. 

This push on Gimme will be balanced with the scope below:

### Current scope

**Revenue generating**
    - yearn.fi
    - Juiced Vaults ($2.5m TVL in a week)
    - OP staking 
    - yCRV
    - yBribe
    - veYFI
    - yAudit
    - V3
    - yETH
    
**New**
    - Gimme
    - Juiced
    - yPrisma
    
**Infrastructure**
- Web lib | Web Template
- yDaemon (meta, APR, etc.)

**Strategic**
- B2B Dashboard
- ape.tax
- yearn.clinic
- NFTreasury.click
- Macarena.finance

### The Plan
As always, the Xopowo team remains committed to both maintaining and improving Yearn's current front ends while also looking to create products that bring new users to Yearn through innovative approaches and new functionality. Ensuring a great product experience for current users while seeking new ones. We therefore propose that the Xopowo team continues maintaining Yearn's website, in addition to creating new projects.

Additionally, the team is going to be searching for additional opportunities for product development, partnerships and integrations with other projects and protocols. This may involve grants or funding opportunities from external sources to Yearn - which will bring value back to Yearn through integrations and complementary functions that enhance the user experience. 

In the future, the Xopowo Team may seek to reduce its dependence on grants from Yearn and move towards a revenue-sharing scheme. The team believes that this approach will help align its incentives with Yearn's interests while also promoting sustainable growth.

### Deadline

2023-04-31

### Money
We propose a dual approach to funding contributions toward the development and maintenance of our projects. Firstly, we plan to distribute a grant to each person working on a project.

Secondly, we will use bounties to fund isolated contributions or contributions linked to projects in maintenance mode. This approach will enable us to incentivize individual contributors, ensuring the continued success and growth of our projects.

Our breakdown:
| Description  | Token | Amount |
| ------------ | ----- | ------ |
| Contributors | DAI   | 95000  |
| Infra        | DAI   | 2000   | 

### Amount

-   97,000 DAI per month

### Wallet address

0x4F909396A75FE9d59F584156A851B3770f3F438a

### Reporting

Monthly

### Plan

Xopowo's overall commitment remains the same as in #168. In short: Xopowo is focused on supporting existing Yearn projects while innovating and shipping new ones.

### Highlights

- Built and launched a flagship UI for Yearn v3.
- yPrisma UI v2 hitting prod in the coming days.
- Gimme (a new Yearn front end) in design phase with the team very excited about its potential.
- Juiced Vaults, a new brand for high reward vaults, built and launched with iterations and improvements to come (not to mention new Vaults).
- Several iterations on what and how to show APRs on V3
- yDaemon: better prices, better performance, better data
- Halfway through code split to make all Yearn dapps easier to maintain.
- The team grew with new devs added and we are still looking to add another dev and UI designer. This was an area of weakness in our last BR and we are happy with the progress here.

**tldr**
The last quarter saw a big push on new launches like v3, yPrisma, and Juiced Vaults. Giving each Yearn project it's own visual brand with bespoke UI design is certainly more labour intensive but we feel definitely worth the effort (with the response to the recent Juiced launch being particularily encouraging for this strategy) and we're excited to share the yPrisma UI refresh soon. 

Next quarter we are excited to build Gimme which we see as a big opportunity for Yearn and strategically significant on multiple fronts. 

This push on Gimme will be balanced with the scope below:

### Current scope

**Revenue generating**
    - yearn.fi
    - Juiced Vaults ($2.5m TVL in a week)
    - OP staking 
    - yCRV
    - yBribe
    - veYFI
    - yAudit
    - V3
    - yETH
    
**New**
    - Gimme
    - Juiced
    - yPrisma
    
**Infrastructure**
- Web lib | Web Template
- yDaemon (meta, APR, etc.)

**Strategic**
- B2B Dashboard
- ape.tax
- yearn.clinic
- NFTreasury.click
- Macarena.finance

### The Plan
As always, the Xopowo team remains committed to both maintaining and improving Yearn's current front ends while also looking to create products that bring new users to Yearn through innovative approaches and new functionality. Ensuring a great product experience for current users while seeking new ones. We therefore propose that the Xopowo team continues maintaining Yearn's website, in addition to creating new projects.

Additionally, the team is going to be searching for additional opportunities for product development, partnerships and integrations with other projects and protocols. This may involve grants or funding opportunities from external sources to Yearn - which will bring value back to Yearn through integrations and complementary functions that enhance the user experience. 

In the future, the Xopowo Team may seek to reduce its dependence on grants from Yearn and move towards a revenue-sharing scheme. The team believes that this approach will help align its incentives with Yearn's interests while also promoting sustainable growth.

### Deadline

2023-04-31

### Money
We propose a dual approach to funding contributions toward the development and maintenance of our projects. Firstly, we plan to distribute a grant to each person working on a project.

Secondly, we will use bounties to fund isolated contributions or contributions linked to projects in maintenance mode. This approach will enable us to incentivize individual contributors, ensuring the continued success and growth of our projects.

Our breakdown:
| Description  | Token | Amount |
| ------------ | ----- | ------ |
| Contributors | DAI   | 95000  |
| Infra        | DAI   | 2000   | 

### Amount

-   97,000 DAI per month

### Wallet address

0x4F909396A75FE9d59F584156A851B3770f3F438a

### Reporting

Monthly

### Deadline

2023-03-03

### People

Xopowo's overall commitment remains the same as in #168. In short: Xopowo is focused on supporting existing Yearn projects while innovating and shipping new ones.

### Highlights

- Built and launched a flagship UI for Yearn v3.
- yPrisma UI v2 hitting prod in the coming days.
- Gimme (a new Yearn front end) in design phase with the team very excited about its potential.
- Juiced Vaults, a new brand for high reward vaults, built and launched with iterations and improvements to come (not to mention new Vaults).
- Several iterations on what and how to show APRs on V3
- yDaemon: better prices, better performance, better data
- Halfway through code split to make all Yearn dapps easier to maintain.
- The team grew with new devs added and we are still looking to add another dev and UI designer. This was an area of weakness in our last BR and we are happy with the progress here.

**tldr**
The last quarter saw a big push on new launches like v3, yPrisma, and Juiced Vaults. Giving each Yearn project it's own visual brand with bespoke UI design is certainly more labour intensive but we feel definitely worth the effort (with the response to the recent Juiced launch being particularily encouraging for this strategy) and we're excited to share the yPrisma UI refresh soon. 

Next quarter we are excited to build Gimme which we see as a big opportunity for Yearn and strategically significant on multiple fronts. 

This push on Gimme will be balanced with the scope below:

### Current scope

**Revenue generating**
    - yearn.fi
    - Juiced Vaults ($2.5m TVL in a week)
    - OP staking 
    - yCRV
    - yBribe
    - veYFI
    - yAudit
    - V3
    - yETH
    
**New**
    - Gimme
    - Juiced
    - yPrisma
    
**Infrastructure**
- Web lib | Web Template
- yDaemon (meta, APR, etc.)

**Strategic**
- B2B Dashboard
- ape.tax
- yearn.clinic
- NFTreasury.click
- Macarena.finance

### The Plan
As always, the Xopowo team remains committed to both maintaining and improving Yearn's current front ends while also looking to create products that bring new users to Yearn through innovative approaches and new functionality. Ensuring a great product experience for current users while seeking new ones. We therefore propose that the Xopowo team continues maintaining Yearn's website, in addition to creating new projects.

Additionally, the team is going to be searching for additional opportunities for product development, partnerships and integrations with other projects and protocols. This may involve grants or funding opportunities from external sources to Yearn - which will bring value back to Yearn through integrations and complementary functions that enhance the user experience. 

In the future, the Xopowo Team may seek to reduce its dependence on grants from Yearn and move towards a revenue-sharing scheme. The team believes that this approach will help align its incentives with Yearn's interests while also promoting sustainable growth.

### Deadline

2023-04-31

### Money
We propose a dual approach to funding contributions toward the development and maintenance of our projects. Firstly, we plan to distribute a grant to each person working on a project.

Secondly, we will use bounties to fund isolated contributions or contributions linked to projects in maintenance mode. This approach will enable us to incentivize individual contributors, ensuring the continued success and growth of our projects.

Our breakdown:
| Description  | Token | Amount |
| ------------ | ----- | ------ |
| Contributors | DAI   | 95000  |
| Infra        | DAI   | 2000   | 

### Amount

-   97,000 DAI per month

### Wallet address

0x4F909396A75FE9d59F584156A851B3770f3F438a

### Reporting

Monthly

### Money

Xopowo's overall commitment remains the same as in #168. In short: Xopowo is focused on supporting existing Yearn projects while innovating and shipping new ones.

### Highlights

- Built and launched a flagship UI for Yearn v3.
- yPrisma UI v2 hitting prod in the coming days.
- Gimme (a new Yearn front end) in design phase with the team very excited about its potential.
- Juiced Vaults, a new brand for high reward vaults, built and launched with iterations and improvements to come (not to mention new Vaults).
- Several iterations on what and how to show APRs on V3
- yDaemon: better prices, better performance, better data
- Halfway through code split to make all Yearn dapps easier to maintain.
- The team grew with new devs added and we are still looking to add another dev and UI designer. This was an area of weakness in our last BR and we are happy with the progress here.

**tldr**
The last quarter saw a big push on new launches like v3, yPrisma, and Juiced Vaults. Giving each Yearn project it's own visual brand with bespoke UI design is certainly more labour intensive but we feel definitely worth the effort (with the response to the recent Juiced launch being particularily encouraging for this strategy) and we're excited to share the yPrisma UI refresh soon. 

Next quarter we are excited to build Gimme which we see as a big opportunity for Yearn and strategically significant on multiple fronts. 

This push on Gimme will be balanced with the scope below:

### Current scope

**Revenue generating**
    - yearn.fi
    - Juiced Vaults ($2.5m TVL in a week)
    - OP staking 
    - yCRV
    - yBribe
    - veYFI
    - yAudit
    - V3
    - yETH
    
**New**
    - Gimme
    - Juiced
    - yPrisma
    
**Infrastructure**
- Web lib | Web Template
- yDaemon (meta, APR, etc.)

**Strategic**
- B2B Dashboard
- ape.tax
- yearn.clinic
- NFTreasury.click
- Macarena.finance

### The Plan
As always, the Xopowo team remains committed to both maintaining and improving Yearn's current front ends while also looking to create products that bring new users to Yearn through innovative approaches and new functionality. Ensuring a great product experience for current users while seeking new ones. We therefore propose that the Xopowo team continues maintaining Yearn's website, in addition to creating new projects.

Additionally, the team is going to be searching for additional opportunities for product development, partnerships and integrations with other projects and protocols. This may involve grants or funding opportunities from external sources to Yearn - which will bring value back to Yearn through integrations and complementary functions that enhance the user experience. 

In the future, the Xopowo Team may seek to reduce its dependence on grants from Yearn and move towards a revenue-sharing scheme. The team believes that this approach will help align its incentives with Yearn's interests while also promoting sustainable growth.

### Deadline

2023-04-31

### Money
We propose a dual approach to funding contributions toward the development and maintenance of our projects. Firstly, we plan to distribute a grant to each person working on a project.

Secondly, we will use bounties to fund isolated contributions or contributions linked to projects in maintenance mode. This approach will enable us to incentivize individual contributors, ensuring the continued success and growth of our projects.

Our breakdown:
| Description  | Token | Amount |
| ------------ | ----- | ------ |
| Contributors | DAI   | 95000  |
| Infra        | DAI   | 2000   | 

### Amount

-   97,000 DAI per month

### Wallet address

0x4F909396A75FE9d59F584156A851B3770f3F438a

### Reporting

Monthly

### Amount

100

### Wallet address

0x0

### Reporting

Once
